Modules/HandshakeMainModule.php

```diff
@@ -0,0 +1,61 @@
+<?php declare(strict_types = 1);
+
+/*  Copyright (c) 2023 Nikita Zhavoronkov, nikzh@nikzh.com
+ *  Copyright (c) 2023 3xpl developers, 3@3xpl.com
+ *  Distributed under the MIT software license, see the accompanying file LICENSE.md  */
+
+/*  This is the main Handshake module. It requires an HSD node (https://github.com/handshake-org/hsd)
+ *  with `index-tx` set to 1 to run.  */
+
+final class HandshakeMainModule extends HandshakeLikeMainModule implements Module
+{
+    function initialize()
+    {
+        // CoreModule
+        $this->blockchain = 'handshake';
+        $this->module = 'handshake-main';
+        $this->is_main = true;
+        $this->currency = 'handshake';
+        $this->currency_details = ['name' => 'Handshake', 'symbol' => 'HNS', 'decimals' => 8, 'description' => null];
```

Comment:
Haven't read everything, noticed this while skimming: `decimals` for HNS is 6, not 8

Reply:
Thanks, that's already been fixed in production, but you were faster to notice this here 😅